crates/bevy_ecs/src/change_detection.rs

```diff
@@ -51,12 +51,22 @@ pub trait DetectChanges {
     /// Returns `true` if this value was added after the system last ran.
     fn is_added(&self) -> bool;
 
+    /// Returns `true` if this value was mutably dereferenced either since
+    /// the last time the system ran or, if the system never ran,
```

Comment:
This line is misleading: if the system has never been run, is_added will always be true.